src/YetORM/Reflection/EntityType.php

```diff
@@ -120,83 +117,110 @@ private function getClassTree()
 		return array_reverse($tree);
 	}
 
+	/**
+	 * Returns an annotation value.
+	 */
+	public static function parseAnnotation(\Reflector $ref, string $name): ?string
+	{
+		if (!NReflection::areCommentsAvailable()) {
+			throw new Nette\InvalidStateException('You have to enable phpDoc comments in opcode cache.');
+		}
+		$re = '#[\s*]@' . preg_quote($name, '#') . '(?=\s|$)(?:[ \t]+([^@\s]\S*))?#';
+		if ($ref->getDocComment() && preg_match($re, trim($ref->getDocComment(), '/*'), $m)) {
+			return $m[1] ?? '';
+		}
+		return null;
+	}
+
 
 	/**
 	 * @param  string $class
 	 * @return void
 	 */
-	private static function loadAnnotationProperties($class)
+	private static function loadAnnotationProperties($class): void
 	{
 		if (!isset(self::$annProps[$class])) {
 			self::$annProps[$class] = [];
-			$ref = $class::getReflection();
-
-			foreach ($ref->getAnnotations() as $ann => $values) {
-				if ($ann === 'property' || $ann === 'property-read') {
-					foreach ($values as $line) {
-						$matches = NStrings::match($line, '#^[ \t]*(?P<type>\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*(?:\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)*(?:\|\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*(?:\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)*)?)[ \t]+(?P<property>[\$a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)(?:[ \t]+->[ \t]+(?P<column>[a-zA-Z0-9_-]+))?[ \t]*(?P<description>.*)\z#');
+			preg_match_all('/(?m)@(\S+) (\S+) (\S+(?: -> \S+)*)(.+)*$/', ($class::getReflection())->getDocComment(), $matches, PREG_SET_ORDER, 0);
```

Comment:
Why still not use `NStrings::matchAll()` ?

Reply:
Yeah, good idea. We wanna fix small bug with `*/` on the end of line - now should be fixed and set to NStrings.